packages/venia-concept/webpack.config.js

```diff
@@ -9,7 +9,7 @@ module.exports = async env => {
     getMediaURL().then(mediaURL => {
         global.MAGENTO_MEDIA_BACKEND_URL = mediaURL;
     });
-    const config = await configureWebpack({
+    const { clientConfig, serviceWorkerConfig } = await configureWebpack({
```

Comment:
This will be a breaking change for developers who are already using Venia since the signature of `configureWebpack` has been changed to accommodate parallel child webpack compilations. This might be useful for our vendors who might want to have multiple compilations not just for Service Worker. This can be used as an API 🙈

Reply:
This solution is scalable since it is an object. We and vendors have full control over adding multiple webpack configs. Not ideally done but they can 2 versions of the same config 1 for development and other for production both built with 1 build command if they want. Also, we can have 2 build configurations, 1 for normal operation and other for SSR output or lite version of the same bundle for mobiles etc. The possibilities are endless.

Reply:
> This solution is scalable since it is an object.

D'oh! 😛 

I guess I was thinking about the return value below (`return [clientConfig, serviceWorkerConfig]`), but modifying that isn't a breaking change so no concern there 👍

Reply:
Oh thought you were talking about line 12. Got it 👍